custom_components/myskoda/translations/en.json

```diff
@@ -198,6 +198,14 @@
             "window_heating": {
                 "name": "Window Heating"
             }
+        },
+        "button": {
+            "honk": {
+                "name": "Honk"
```

Comment:
We send HONK_AND_FLASH to Myskoda API when calling this.Does this result in "just" a Honk in reality?